qpicard/src/org/qcmg/picard/CramFileReaderFactory.java

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
don't need this file - can read CRAM files using existing classes with the relevant java option set. Also, if we did want to go down the route of not having to set the java option, we should be extending the existing classes

Reply:
deleted